README.md

```diff
@@ -35,30 +40,36 @@ Environment variables
 
 The following environment variables are exposed for InfraMock:
 
-`EXPERIMENT_ID` : set the experiment ID, or leave unset for default experiment.
+`BIOMAGE_DATA_PATH`: where to get the experiment data to populate inframock's S3 and DynamoDB. If
+this is not set, it will default to `./data`.
 
-`POPULATE_MOCK`: whether localstack should be filled with a mocked PBMC data set. This is
-set to `true` by default, which means that InfraMock will try to use a locally running version of
-the `api` to populate the localstack DynamoDB database with using its `/experiments/generate` endpoint.
-For this to work, `api` **must** be running with `CLUSTER_ENV` set to `development`, which is the default behavior.
-
-`MOCK_EXPERIMENT_DATA_PATH`: where to get the mocked data for upload to the local S3 from. If
-this is not set, it will default to the Github URL where the dataset used for the `worker` unit
-test is located.
+`POPULATE_MOCK`: whether localstack should be filled with the data sets found in `BIOMAGE_DATA_PATH`.
```

Comment:
what happens if the user sets `POPULATE_MOCK` to `false`?

Reply:
it will just spin up localstack services without adding any data to S3 / DynamoDB

Reply:
It was like this already, because in the beginning, we had it configure to be able to use real DynamoDB and post messages to the worker in production. In practice, we never used this approach, because there is no way to receive a message back to the production worker when in inframock and this is because of the SNS topic subscription URL. Even if we send a message locally to the production worker, the production worker will send it back to the production api.

Reply:
This is back from the time when we didn't have staging -- we only had local development and production